### PR TITLE
Revert "Merge pull request #373 from MoveOnOrg/alex-tech-secrets"

### PR DESCRIPTION
**Pull request by technicalex**
This pull request is the first step towards updating the main branch with the working code already deployed to Lambda. The fix involved backing out the changes made in the previous pull request, so I am doing that here in a separate pull request for clarity. A subsequent pull request will correctly update the script to use AWS Secrets Manager.

These code changes were generated automatically using `git revert -m 1`.

This reverts commit 223a8e37201e49e48d0e2d72981992c09a5e5228, reversing
changes made to 94ebdfbb24949fb638c9cac6892c745c08d41e47.

### CODE REVIEW

**Comment by technicalex** [comment]
@ibrand has already seen these changes, as I made them while pairing. @codygordon and @sjwmoveon, you are added mostly for awareness.

File: README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ This is designed to handle failure gracefully. If processing of a survey fails,
 
 ActionKit tables can be in a separate schema, but currently must be accessible within the same database connection.
 
-## Settings
+## settings.py
 
-Settings are fetched directly from AWS Secrets Manager using the secret name `ak-survey-results`.
```
Comment (ibrand) [inline README.md:21]: quick question: did this change get reverted on purpose or by accident? When we were pairing, we were working off of the settings.py file, but I know these changes involve migrating to secrets manager

**Comment by ibrand** [comment]
Ah ok, so this is just the PR to revert back to "main before your last change" and then the next PR will do the update to the code for secrets manager?

**Comment by technicalex** [comment]
@ibrand, yes, exactly! A few of these changes will be re-instated in the next PR, but since the vast majority were removed, I thought it would be cleaner to back out everything and add back only what was necessary.

**Review by ibrand — APPROVED** [review]

**Pull request merged**